PWGDQ/Core/VarManager.h

```diff
@@ -14,13 +14,13 @@
 // Class to handle analysis variables
 //
 
+#ifndef PWGDQ_CORE_VARMANAGER_H_
+#define PWGDQ_CORE_VARMANAGER_H_
+
 #ifndef HomogeneousField
 #define HomogeneousField
 #endif
 
-#ifndef PWGDQ_CORE_VARMANAGER_H_
-#define PWGDQ_CORE_VARMANAGER_H_
-
```

Comment:
@iarsene We were forced to make this change for passing MegaLinter checks
We believe it is safe but, please confirm we didn't make anything incorrect